### PR TITLE
Update Cromha Expansion.yaml

### PR DESCRIPTION
**Pull request by OcelotWalrus**
Use Continuous Build instead of releases.

I made so it still update the version name.

### CODE REVIEW

**Comment by OcelotWalrus** [comment]
Since my Continuous Build refresh at every push, there is no difference with the commit tracking stuff.

**Comment by MCOfficer** [comment]
I would still prefer a commit-based manifest. The point of PRs is that changes can be reviewed before they're merged.

**Comment by OcelotWalrus** [comment]
Sure! I made it commit based.

I just have a problem, how do I do for the download url? Do I make it the continuous url?

File: manifests/Cromha Expansion.yaml [hunk 1]
```diff
@@ -11,9 +11,10 @@ description: 'Adds a new civilization to the Korath Space (now named the ''Empir
...
+url: https://github.com/OcelotWalrus/Cromha-Expansion-plugin/releases/download/9.9.9-continuous/Cromha-Expansion-plugin.zip
+iconUrl: https://github.com/OcelotWalrus/Cromha-Expansion-plugin/blob/a5c45028d4fe2854ce13fb3109b05d8226cf0015/icon.png?raw=true
 autoupdate:
-  type: tag
-  url: https://github.com/OcelotWalrus/Cromha-Expansion-plugin/releases/download/$version/Cromha-Expansion-plugin.zip
-  iconUrl: https://github.com/OcelotWalrus/Cromha-Expansion-plugin/blob/$version/icon.png?raw=true
+  type: commit
+  url: https://github.com/OcelotWalrus/Cromha-Expansion-plugin/releases/download/9.9.9-continuous/Cromha-Expansion-plugin.zip
+  iconUrl: https://github.com/OcelotWalrus/Cromha-Expansion-plugin/raw/$version/icon.png
+
```
Comment (MCOfficer) [inline manifests/Cromha Expansion.yaml:20]:
```suggestion
```

**Comment by OcelotWalrus** [comment]
@CMOfficer could you merge it right now? Because I want to commit to my repository without having to update my manifest file.

File: manifests/Cromha Expansion.yaml [hunk 2]
```diff
@@ -11,9 +11,9 @@ description: 'Adds a new civilization to the Korath Space (now named the ''Empir
   They are human but don''t directly have contact with human Merchants, they have
   contact with the Syndicate, the Republic but they don''t allow them to enter their
   territory. Know more about the plugin in the wiki! (or test the game yourself)'
-url: https://github.com/OcelotWalrus/Cromha-Expansion-plugin/releases/download/v0.0.2/Cromha-Expansion-plugin.zip
-iconUrl: https://github.com/OcelotWalrus/Cromha-Expansion-plugin/blob/v0.0.2/icon.png?raw=true
+url: https://github.com/OcelotWalrus/Cromha-Expansion-plugin/releases/download/9.9.9-continuous/Cromha-Expansion-plugin.zip
```
Comment (MCOfficer) [inline manifests/Cromha Expansion.yaml:14]:
```suggestion
url: https://github.com/OcelotWalrus/Cromha-Expansion-plugin/archive/92f5020b0cbfb7f3dbfe80900ac8ae9c6eaaf482.zip
```

File: manifests/Cromha Expansion.yaml [hunk 3]
```diff
@@ -11,9 +11,9 @@ description: 'Adds a new civilization to the Korath Space (now named the ''Empir
...
+iconUrl: https://github.com/OcelotWalrus/Cromha-Expansion-plugin/blob/92f5020b0cbfb7f3dbfe80900ac8ae9c6eaaf482/icon.png?raw=true
 autoupdate:
-  type: tag
-  url: https://github.com/OcelotWalrus/Cromha-Expansion-plugin/releases/download/$version/Cromha-Expansion-plugin.zip
-  iconUrl: https://github.com/OcelotWalrus/Cromha-Expansion-plugin/blob/$version/icon.png?raw=true
+  type: commit
+  url: https://github.com/OcelotWalrus/Cromha-Expansion-plugin/releases/download/9.9.9-continuous/Cromha-Expansion-plugin.zip
```
Comment (MCOfficer) [inline manifests/Cromha Expansion.yaml:18]:
```suggestion
  url: https://github.com/OcelotWalrus/Cromha-Expansion-plugin/archive/$version.zip
```

**Review by MCOfficer — CHANGES_REQUESTED** [review]

**Comment by lumbar527** [comment]
> Will let this sit for a few hours so others have the chance to chime in.

Does that mean wait for other peoples opinion? If so, then I think it makes a little more sense to have the plugin page/ESLauncher display the continuous, rather than release, because it is in beta anyway and new things are added quickly.

**Comment by lumbar527** [comment]
I might be misinterpreting some, but I think it should be done by commit, not regularly updated release.

**Pull request merged**